freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/FreeIpaV1Endpoint.java

```diff
@@ -43,11 +44,15 @@
 
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.Authorization;
 
 @RetryAndMetrics
 @Path("/v1/freeipa")
 @Consumes(MediaType.APPLICATION_JSON)
-@Api(value = "/v1/freeipa", protocols = "http,https", consumes = MediaType.APPLICATION_JSON)
+@Api(value = "/v1/freeipa",
+        protocols = "http,https",
+        authorizations = {@Authorization(value = FreeIpaApi.CRN_HEADER_API_KEY)},
```

Comment:
will it work with DP CLI?

Reply:
I hope so, but will give it a try to be sure. The same approach is already in place for `redbeams`.